airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -217,7 +232,10 @@ def get_upstream_asset_events(
 @dag_run_router.post(
     "/{dag_run_id}/clear",
     responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
-    dependencies=[Depends(action_logging())],
+    dependencies=[
+        Depends(requires_access_dag(method="PUT", access_entity=DagAccessEntity.RUN)),
```

Comment:
Should it be post instead?

```suggestion
        Depends(requires_access_dag(method="POST", access_entity=DagAccessEntity.RUN)),
```

Reply:
https://github.com/apache/airflow/pull/47532#discussion_r1988827346